modules/manage/pages/schema-reg/schema-reg-authorization.adoc

```diff
@@ -0,0 +1,355 @@
+| `subject`
+
+| `/subjects/{subject}/versions/referencedby`
+| `GET`
+| `describe`
+| `global`
+
+| `compatibility/subjects/{subject}/versions/{version}`
+| `POST`
+| `read`
+| `subject`
+
+| `/status/ready`
+| `GET`
+| none/open
+| -
+
+| `/security/acls`
+| `GET`
+| `describe`
+| `cluster_action`
+
+| `/security/acls`
+| `POST`
+| `alter`
+| `cluster_action`
+
+| `/security/acls`
+| `DELETE`
+| `alter`
+| `cluster_action`
+|===
+
+For additional guidance on these operations, see the xref:api:ROOT:schema-registry-api.adoc#get-/security/acls[Redpanda Schema Registry API].
+
+== Enable Schema Registry Authorization
+
+=== Prerequisites
+
+Before you can enable Schema Registry Authorization, you must have:
+
+* A valid Redpanda Enterprise license
+* Cluster administrator permissions to modify cluster configuration
+
+=== Enable authorization
+
+To enable Schema Registry Authorization for your cluster:
+
+[,bash]
+----
+rpk cluster config set schema_registry_enable_authorization true
+----
+
+For details, see xref:reference:properties/cluster-properties.adoc#schema_registry_enable_authorization[`schema_registry_enable_authorization`].
+
+== Create and manage Schema Registry ACLs
+
+This section shows you how to create and manage ACLs for Schema Registry resources.
+
+=== Create an ACL for a topic and Schema Registry subject
+
+This example creates an ACL that allows the principal `panda` to read from both the topic `bar` and the Schema Registry subject `bar-value`. This pattern is common when you want to give a user or application access to both the Kafka topic and its associated schema.
+
+[,bash]
+----
+rpk security acl create --allow-principal panda --operation read --topic bar --registry-subject bar-value
+
+PRINCIPAL   HOST  RESOURCE-TYPE  RESOURCE-NAME  RESOURCE-PATTERN-TYPE  OPERATION  PERMISSION  ERROR
+User:panda  *     SUBJECT        bar-value      LITERAL                READ       ALLOW
+User:panda  *     TOPIC          bar            LITERAL                READ       ALLOW
+
+----
+
+=== Create an ACL for global Schema Registry access
+
+This example grants the user `jane` global read and write access to the Schema Registry, plus read and write access to the topic `private`. The `--registry-global` flag creates ACLs for all global Schema Registry operations.
+
+[,bash]
+----
+rpk security acl create --allow-principal jane --operation read,write --topic private --registry-global
+PRINCIPAL  HOST  RESOURCE-TYPE  RESOURCE-NAME  RESOURCE-PATTERN-TYPE  OPERATION  PERMISSION  ERROR
+User:jane  *     REGISTRY                      LITERAL                READ       ALLOW
+User:jane  *     REGISTRY                      LITERAL                WRITE      ALLOW
+User:jane  *     TOPIC          private        LITERAL                READ       ALLOW
+User:jane  *     TOPIC          private        LITERAL                WRITE      ALLOW
+----
+
+User `jane` now has global `read` and `write` access to the Schema Registry, and to the topic `private`.
```

Comment:
This, for me, implies Global = All access. It might be worth linking to the table above, which shows what READ and WRITE for REGISTRY actually gives access to